docs/ENV.md

```diff
@@ -17,7 +17,13 @@ In addition to the application configuration settings described in [Configuring
     * The device client will also print the contents of the trace during shutdown when memory tracing is enabled.
     * When there are no pending allocations or memory trace is not enabled, then nothing is printed to the SDK log file.
     * Enabling memory allocation tracing has a nontrivial cost and we do not recommend that customers enable this by default for production deployments.
-
+    
+* `LOCK_FILE_PATH`
+  * To enforce single instance creation, set `LOCK_FILE_PATH` to a file in a writeable directory. For example, if you want to write to a file named devicecl.lock in /run/lock, then `LOCK_FILE_PATH=/run/lock/devicecl.lock`. Permissions still apply when writing to restricted directories.
+  * This feature is disabled until the environment variable is set.
```

Comment:
How is this true if we have a default value set for `LOCK_FILE_PATH`?

Reply:
forgot to update the documentation, sorry